standard_reports/expiring-items/latest/convert_data_js/src/convert_data.js

```diff
@@ -1,16 +1,13 @@
-import { processStockLines } from "./utils";
+import { processStockLines } from './utils';
 
-function convert_data() {
-  let res = JSON.parse(Host.inputString());
+function convert_data(res) {
```

Comment:
all of these should be safe, no logical differences, please let me know if you see anything